pages/inventory.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <v-container>
+    <v-layout wrap>
+      <v-flex xs10 sm6 offset-sm3 offset-xs1>
+        <v-card class="mt-0 mb-4" color="#C1FFC1">
+          <v-card-text>
+            <div>
+              <h1 class="inventory-header">
+                Inventory
+              </h1>
+            </div>
+          </v-card-text>
+        </v-card>
+      </v-flex>
+    </v-layout>
+    <v-layout row wrap>
+      <v-card
+        v-for="inven in inventory.microcontroller"
+        :key="inven['.key']"
+        class="mx-auto mb-3"
+        width="400"
+      >
+        <v-flex xs12 class="text-xs-center">
+          <v-avatar :tile="false" :size="200" color="grey lighten-4 mt-3">
+            <v-img :aspect-ratio="16 / 9" :src="inven.thumbnail" alt="Avatar" />
```

Comment:
@irfankhaan There isn't any thumbnail key in the database.

Reply:
I was thinking I will add the thumbnail when I have to make a json for the actual inventory data. What do you say?

Reply:
No need of thumbnail. Use DataTable component for the invrntory